launchflow/workflows/deploy_gcp_service.py

```diff
@@ -17,13 +17,20 @@
 from launchflow.config import config
 from launchflow.gcp.cloud_run import CloudRun
 from launchflow.gcp.compute_engine_service import ComputeEngineService
+from launchflow.gcp.gke_service import GKEService
 from launchflow.gcp.firebase_site import FirebaseStaticSite
 from launchflow.gcp.service import GCPDockerService
 from launchflow.gcp.static_site import StaticSite
 from launchflow.managers.service_manager import ServiceManager
 from launchflow.models.flow_state import GCPEnvironmentConfig, ServiceState
+from launchflow.workflows.k8s_service import update_k8s_service
 from launchflow.workflows.utils import tar_source_in_memory
 
+from kubernetes import config as k8_config
+
+if TYPE_CHECKING:
+    from google.cloud.container import Cluster
```

Comment:
I think this needs to be in a try catch to handle the case where user hasnt installed launchflow[gcp]

Reply:
type checking won't be set for regular installs this will only try and import it during a special type checking mode that type checkers use